Add meta descriptions to Lisk Core docs, closes #371 (#393)
LiskHQ/lisk-docs: modules/ROOT/pages/getting-started/maintain-a-node.adoc

= How to maintain a node
Mona Bärenfänger <mona@lightcurve.io>

> :description: Learn how to maintain a Lisk node and in what cases it is recommended to run an own node.

Reviewer: ```suggestion
:description: Learn how to maintain a Lisk node and also when it is recommended to run a specific node.
```
